<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VC_ui</class>
 <widget class="QWidget" name="VC_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>253</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>644</width>
    <height>253</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>644</width>
    <height>253</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Вершинное покрытие</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>248</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="inputPath"/>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="inputButton">
      <property name="text">
       <string>Открыть</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="showButton">
      <property name="text">
       <string>Показать</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>_</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="modeLabel">
      <property name="text">
       <string>Режим</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="modeCombo">
      <item>
       <property name="text">
        <string>полный перебор</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>двоичный поиск по k-подмножествам</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>через клику с быстрым отсечением</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>Запуск</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="outputButton">
      <property name="text">
       <string>Вывод</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="outputPath"/>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="approxLimitBox">
        <property name="text">
         <string>Определить границы аппроксимацией</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
